### PR TITLE
Modified visit model

### PR DESCRIPTION
**Pull request by hsorellana**
This is a correction of the visit model. I had to open a new PR since i deleted the fork and all changes locally made are gone.

I suggest the PR #919 should be deleted since its commit are in this one, and i can keep tracking the work here.
cc @HospitalRun/core-maintainers

### CODE REVIEW

File: app/models/visit.js [hunk 1]
```diff
@@ -37,10 +37,8 @@ function paymentStateAcceptance(object) {
...
 
 export default AbstractModel.extend({
-  additionalDiagnoses: DS.attr(), // Yes, the plural of diagnosis is diagnoses!
-  charges: DS.hasMany('proc-charge', {
-    async: false
-  }),
+  // Attributes
+  additionalDiagnoses: DS.attr(),
   customForms: DS.attr('custom-forms'),
   diagnoses: DS.hasMany('diagnosis', { async: false }),
```
Comment (jkleinsc) [inline app/models/visit.js]: This line shouldn't be here any more.

**Comment by tangollama** [comment]
@hsorellana thanks for your patience. I'm going to try to merge and close this out today.

**Comment by tangollama** [comment]
@hsorellana it looks like Travis is failing this branch b/c lines 42 and 63 in the visit object are repeating the diagnoses declaration. Could you resolve this? I'd love to get this change into the master.

**Comment by hsorellana** [comment]
@tangollama nice! green light from travis 😄

**Comment by tangollama** [comment]
Awesome. I'm going to confirm with @jkleinsc one more time and the merge. Thank you for the prompt reply!

**Pull request merged**